app/views/layouts/nav/staff/_website_subnav.html.haml

```diff
@@ -1,10 +1,13 @@
 .navbar.navbar-fixed-top.website-subnav
   .container-fluid
-    %ul.nav.navbar-nav.navbar-right
-      %li{class: subnav_item_class('event-pages-link')}
-        = link_to event_staff_pages_path(current_event) do
-          %span Pages
     %ul.nav.navbar-nav.navbar-right
       %li{class: subnav_item_class('event-website-configuration-link')}
         = link_to edit_event_staff_website_path(current_event) do
           %span Configuration
+      %li{class: subnav_item_class('event-pages-link')}
+        = link_to event_staff_pages_path(current_event) do
+          %span Pages
+      %li{class: subnav_item_class('event-sponsor-link')}
+        = link_to event_staff_sponsors_path do
+          %span Sponsors
+          %sup Beta
```

Comment:
We probably don't need the Beta on Sponsors now that it is in the Website subnav.

Reply:
👍